integration/tests/cli/message/test_message_send_command.rs

```diff
@@ -94,8 +99,21 @@ impl TestMessageSendCmd {
             TestTopicId::Named => self.topic_name.clone(),
         });
 
+        if let Some(header) = &self.header {
+            command.push("-H".to_string());
+            command.push(
+                header
+                    .iter()
+                    .map(|(k, v)| format!("{}:{}:{}", k, v.kind, v.value_only_to_string()))
```

Comment:
`k` can be included in format string

Reply:
I don't get that one. `k` is part of the format string, isn't it? Can you explain it again?

Reply:
it's possible to do like this `format!("{k}...`